app/bundles/LeadBundle/Resources/views/Company/list_rows_contacts.html.twig

```diff
@@ -192,7 +192,11 @@
                     </td>
                     <td>
                         <a href="{{ path('mautic_contact_action', {'objectAction': 'view', 'objectId': contact.id}) }}" data-toggle="ajax">
-                            <div>{% if contact.isAnonymous %}{{ contact.primaryIdentifier|trans }}{% else %}{{ contact.primaryIdentifier|purify }}{% endif %}</div>
+                            {% set primaryIdentifier = contact.isAnonymous ? contact.primaryIdentifier|trans : contact.primaryIdentifier|purify %}
+                            <div>
+                                <span>{{ primaryIdentifier }}</span><br>
```

Comment:
I would avoid use <br /> tag, probably <span> tag either.
Only add customContent is enought